packages/hs-react-ui/src/components/Text/Text.stories.tsx

```diff
@@ -44,8 +47,8 @@ export const Default = () => (
   <Text
     size={text('size', '1rem')}
     color={color('color', colors.grayXlight)}
-    iconPrefix={select('iconPrefix', options, options.mdiComment)}
-    iconSuffix={select('iconSuffix', options, options.mdiComment)}
+    iconPrefix={getIconPrefixPath(select('iconPrefix', options, options.mdiComment))}
+    iconSuffix={getIconSuffixPath(select('iconPrefix', options, options.mdiComment))}
```

Comment:
looks like this change breaks the iconSuffix knob
```suggestion
    iconSuffix={getIconSuffixPath(select('iconSuffix', options, options.mdiComment))}
```